### PR TITLE
Cepheid guide

### PR DESCRIPTION
**Pull request by przemekmroz**
First version of the Cepheid guide

### CODE REVIEW

File: doc/field_guide__cepheid.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+#### Light curve characteristics
+- periodic variable
+- period range: 1-100 days (fundamental-mode pulsators), 0.24-8 days (first-overtone pulsators)
+- amplitude: moderate amplitude (~0.3 to 0.8 mag)
+- light curve shape:
+    - smooth, sawtooth variations (blue/green filters show stronger sawtooth pattern); steep rise and slow decay
+    - Classical Cepheids with pulsation periods 6-20 days may have a secondary bump
+
+#### Other characteristics and selection methods
+- intrinsic Cepheids colors: Bp-Rp = 0.8 -- 2.0 mag, but as these stars are located toward the Galactic plane, the reddening may be significant
```
Comment (mcoughlin) [inline doc/field_guide__cepheid.md:30]: "near" the Galactic plane?

**Review by mcoughlin — CHANGES_REQUESTED** [review]
Maybe _static/taxonomy.html should be a second PR? Seems unrelated. Anyways, could you post a screenshot so we can see the tooltips and such?

Also, we should not be actually committing the the data/*.png 's I think, as those are done on the fly. @dmitryduev should confirm.

File: doc/field_guide__cepheid.md (hunk 1 above)
Comment (przemekmroz) [reply doc/field_guide__cepheid.md:30]: A screenshot of what?

Comment (mcoughlin) [reply doc/field_guide__cepheid.md:30]: Ah nevermind, I see what happened. You can delete taxonomy.html  as well. That is auto-generated too.

**Review by dmitryduev — COMMENTED** [review]
Looks good, just please address what @mcoughlin pointed out: rm the png's and taxonomy.html with `git rm --cached doc/_static/taxonomy.html doc/data/*.png`.

**Comment by przemekmroz** [comment]
Okay. I removed *png and taxonomy.html and changed toward --> near in the description

**Review by dmitryduev — APPROVED** [review]
👍🏻

**Review by mcoughlin — APPROVED** [review]

**Pull request merged**